### DISCUSSION TITLE
Drop Frames

### DISCUSSION THREAD

**Discussion by rotal** [kingslay/KSPlayer#745] [General]
I found the player is dropping frames for vision pro. actually not only vision pro. 
I built in release build and profile. fps is about 40 instead of 60 for 8k video.
I tried the same video with tracyplayer which is not dropping frames, 60fps.
Any reason cause the opensource version drop frames?

**Comment by kingslay** [comment]
KSPlayer 会分为两个版本: LGPL版本和GPL版本，GPL版本会是LGPL版本的子集，某些功能只会存在LGPL版本。并且bug会优先在LGPL版本修复。如果GPL版本有严重bug的话，那会同步修复bug。LGPL版本是闭源的(可以看到源码)，只有经过授权的人才能访问。我的APP（Tracyplayer）使用的是最新的LGPL版本代码
